examples/kft-v2/scripts/trl_training.py

```
@@ -0,0 +1,790 @@
+        if dataset_dir.exists() and any(dataset_dir.iterdir()):
+            try:
+                full_dataset = load_from_disk(str(dataset_dir))
+                if isinstance(full_dataset, dict):
+                    train_dataset = full_dataset.get('train', full_dataset.get('train[:100]'))
+                    test_dataset = full_dataset.get('test', full_dataset.get('test[:20]'))
+                else:
+                    train_size = min(100, len(full_dataset) - 20)
+                    train_dataset = full_dataset.select(range(train_size))
+                    test_dataset = full_dataset.select(range(train_size, min(train_size + 20, len(full_dataset))))
+
+                return train_dataset, test_dataset
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)